nerdlandbot/commands/space_launches.py

```diff
@@ -0,0 +1,117 @@
+import aiohttp
+import asyncio
+import json
+import os
+import discord
+from datetime import datetime
+from discord.ext import commands
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+
+from nerdlandbot.helpers.constants import \
+                                        THE_SPACE_DEVS_BASE_URL, THE_SPACE_DEVS_VERSION, THE_SPACE_DEVS_LIMIT_TO_10_RESULTS, \
+                                        THE_SPACE_DEVS_UPCOMING_LAUNCH_RESOURCE, \
+                                        THE_SPACE_DEVS_LOCAL_TEST_SERVER_BASE_URL, \
+                                        THE_SPACE_DEVS_LOCAL_CACHE_FOLDER, THE_SPACE_DEVS_LOCAL_CACHE_SPACE_LAUNCHES_FILE, \
```

Comment:
It seems that these two are always used together, why not merge them?

Reply:
Personal preference & split up in anticipation of future :-)